elasticsearch.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"github.com/buger/elastigo/api"
 	"github.com/buger/elastigo/core"
-	"github.com/buger/gor/proto"
```

Comment:
I lean towards avoiding a "hard" (i.e. lots of custom stuff, not merge-able back in to upstream, difficult to pull in improvements from upstream) fork when possible. So yes, we are vulnerable to upstream changes, but I'd rather see how big of a problem that is first before deciding to go it alone. My hunch is that it won't be a problem, since the the code change is very isolated.